test/test.js

```diff
@@ -1,280 +1,100 @@
-var assert = require('assert');
-var request = require('supertest');
-var server = require('../server');
-var httpMocks = require('node-mocks-http');
-var url = 'http://localhost:8080';
+const request = require('supertest');
+const url = 'http://localhost:8080';
 
 describe('Front Page', function () {
-  it('should Load Page', function (done) {
-    request(url)
-    .get('/')
-    .send()
-    .end(function (err, res) {
-      if (err) {
-        throw err;
-      }
-      assert.equal(res.statusCode, 200);
-      if (res.text.indexOf("Iron</span>-Fists","Incorrect Page served") > -1) {
-        assert.ok(true)
-      }else{
-        assert.ok(false)
-      }
-      done();
+    it('should Load Page', function (done) {
+        request(url)
+            .get('/')
+            .send()
+            .end(function (err, res) {
+                if (err) {
+                    throw err;
+                }
+                expect.equal(res.statusCode, 200);
+                if (res.text.indexOf('Iron</span>-Fists','Incorrect Page served') > -1) {
+                    expect.to.ok(true);
+                }else{
+                    expect.to.ok(false);
+                }
+                done();
```

Comment:
Might be worth investigating can you use `async` / `await` goodness Frank introduced.

Reply:
ooh separate PR though